judgels-frontends/raphael/src/routes/uriel/contests/ContestRole/ContestRoleColor.tsx

```diff
@@ -0,0 +1,10 @@
+import { Intent } from '@blueprintjs/core';
+
+import { ContestRole } from 'modules/api/uriel/contestWeb';
+
+export const contestRoleColor = {
```

Comment:
This is only used in `ContestRoleTag`, let's not create a separate file for this